src/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -382,11 +381,7 @@ private boolean isAmphibiousBattle() {
   private Collection<TerritoryEffect> getTerritoryEffects() {
     final Collection<TerritoryEffect> territoryEffects = new ArrayList<>();
     if (m_territoryEffectsJList != null) {
-      final List<Object> selectedObjects = Arrays.asList(m_territoryEffectsJList.getSelectedValuesList());
-      final List<String> selected = new ArrayList<>();
-      for (final Object obj : selectedObjects) {
-        selected.add((String) obj);
-      }
+      final List<String> selected = m_territoryEffectsJList.getSelectedValuesList();
```

Comment:
Arrays.asList is used to convert arrays to lists... because `m_territoryEffectsJList.getSelectedValuesList()` returns a List, this list is treated as an object which is very likely to be casted to a `List<Object>` after that the values of that list and then we are iterating through those values casting them back to s String... - my version seems to work fine... No problems - i tested it